oap-server/server-starter/src/main/resources/component-libraries.yml

```diff
@@ -434,6 +434,15 @@ Impala-jdbc-driver:
 Impala:
   id: 134
   languages: Java
+EventMesh:
+  id: 135
+  languages: Java
+eventmesh-producer:
+  id: 136
+  languages: Java
+eventmesh-consumer:
+  id: 137
+  languages: Java
```

Comment:
Would you keep the upper case and lower case consistent? These names seem very strange.

Reply:
OK, if that is not a mistake, I am fine.